test/compilable/b1215.d

```diff
@@ -0,0 +1,118 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS:
```

Comment:
Normally compilable test is compiled with `-c` and generates objfile. But the test for 1215 does not have to do it.
So, you can list `-o-` in `REQUIRED_ARGS:` to stop backend codegen.